src/main/java/be/uantwerpen/minelabs/block/LabBlock.java

```diff
@@ -63,6 +65,7 @@ public BlockState mirror(BlockState state, BlockMirror mirror) {
 
     @Override
     public BlockState getPlacementState(ItemPlacementContext ctx) {
+        // TODO TRIGGER NEIGHBOR UPDATE
```

Comment:
I think neighbour updates are triggered by MC, you don't have to manually call it.